templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+{% block content %}
+  <h1>Delete manufacturer:</h1>
+  <h1>Are you sure about deleting manufacturer info?</h1>
```

Comment:
What if you chose the wrong manufacturer to delete? Maybe it's better to add the manufacturer info?